linker-interface/shared/src/main/scala/org/scalajs/linker/interface/LinkTimeProperties.scala

```diff
@@ -0,0 +1,64 @@
+package org.scalajs.linker.interface
+
+import org.scalajs.ir.Trees._
+
+final class LinkTimeProperties (
```

Comment:
Is there a reason for `LinkTimeProperties` to be in the `interface`? Can't it be kept inside `linker` only?

Reply:
There's no special reason, I just put it in `interface` just because other properties of `CoreSpec` are in `interface`, but there's no reason not to keep inside `linker` only indeed :+1: